clap-utils/src/compute_unit_price.rs

```diff
@@ -1,15 +1 @@
-use {crate::ArgConstant, clap::Arg};
-
-pub const COMPUTE_UNIT_PRICE_ARG: ArgConstant<'static> = ArgConstant {
-    name: "compute_unit_price",
-    long: "--with-compute-unit-price",
-    help: "Set compute unit price for transaction, in increments of 0.000001 lamports per compute unit.",
-};
-
-pub fn compute_unit_price_arg<'a, 'b>() -> Arg<'a, 'b> {
-    Arg::with_name(COMPUTE_UNIT_PRICE_ARG.name)
-        .long(COMPUTE_UNIT_PRICE_ARG.long)
-        .takes_value(true)
-        .value_name("COMPUTE-UNIT-PRICE")
-        .help(COMPUTE_UNIT_PRICE_ARG.help)
-}
+pub use crate::compute_budget::{compute_unit_price_arg, COMPUTE_UNIT_PRICE_ARG};
```

Comment:
One suggestion could be to deprecate the re-export with a message to use them directly from `compute_budget` instead as done [here](https://github.com/anza-xyz/agave/blob/master/clap-v3-utils/src/input_parsers/mod.rs#L20-L28). However, this doesn't really do anything in terms of triggering warnings (https://github.com/solana-labs/solana/pull/33276#discussion_r1336119891), so it is your call.

Reply:
Yeah that was my thinking too, but I guess it can't hurt to document the code. Done!